### ISSUE TITLE
Error experienced when requesting data at midnight

### ISSUE THREAD

**Issue by jacobtomlinson** [jacobtomlinson/datapoint-python#19]
From [log file](https://gist.github.com/jacobtomlinson/2e5abad238ec47b46cd4) for site `Clifton Hill Ski Slope Exeter`

```
Tue Dec  2 21:00:03 2014 - Partly cloudy (night)
Tue Dec  2 22:00:02 2014 - Partly cloudy (night)
Tue Dec  2 23:00:01 2014 - Partly cloudy (night)
Unexpected error: <type 'exceptions.NameError'> on line 64
Wed Dec  3 01:00:01 2014 - Clear night
Wed Dec  3 02:00:03 2014 - Clear night
Wed Dec  3 03:00:04 2014 - Clear night
```

This issue happens every day when requesting data at midnight.

**Comment by NigelRook** [comment]
It appears for ~30-40 minutes after midnight, forecast.days[0] is dated for the previous day. This causes forecast.now() to return False, and possibly other issues.

**Comment by EJEP** [comment]
This is fixed in 0c61a76683672b688447fe7f131c7b31a402c144. The code in forecast_now() checks if the time is shortly after midnight, and if so returns the final forecast
